packages/providers/fal/avatar-generation/fal_avatar/models/kling.py

```
@@ -494,3 +494,194 @@ def get_model_info(self) -> Dict[str, Any]:
+        # Apply defaults
+        character_orientation = character_orientation or self.defaults["character_orientation"]
+        if keep_original_sound is None:
+            keep_original_sound = self.defaults["keep_original_sound"]
+
+        # Validate character_orientation
+        valid_orientations = ["video", "image"]
+        if character_orientation not in valid_orientations:
+            raise ValueError(
+                f"character_orientation must be one of {valid_orientations}, "
+                f"got: '{character_orientation}'"
+            )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)